models/cflow_v2.json

```diff
@@ -0,0 +1,136 @@
+{
+    "fact_table": "cflow_facts_v2",
+    "label": "Cash Flow (V2)",
```

Comment:
Do these new cubes show up in https://municipaldata.treasury.gov.za/ automatically? Do they work? Should we exclude them somehow so we don't show links to broken things until we can make sure they work properly?

Reply:
They do show up. I've added a commit that excludes them from being listed on the data portal.